core/crypto/pedersen_hash.go

```diff
@@ -1,38 +1,84 @@
 package crypto
 
 import (
-	"github.com/NethermindEth/juno/core/crypto/starkware"
+	"math/big"
+
 	"github.com/NethermindEth/juno/core/felt"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve"
```

Comment:
Probably nothing
```suggestion
	starkcurve "github.com/consensys/gnark-crypto/ecc/stark-curve"
```

Reply:
The package name is `starkcurve`, so we don't need to set it manually. Package names [are distinct](https://go.dev/blog/package-names) from package paths (though they are often the same).

Reply:
Yeah. Got you.

The suggestion is more about readability for me

Reply:
I looked into this a bit more and I now think you're right (Uber's Go style guide [agrees](https://github.com/uber-go/guide/blob/master/style.md#import-aliasing) with your suggestion). I'll update the PR.